docs/API/后端API/应用工具类管理.md

```
@@ -0,0 +1,160 @@
+**接口描述：**
+
+
+### 请求参数
+
+**Headers**
+
+| 参数名称     | 参数值           | 是否必须 | 示例 | 备注 |
+| ------------ | ---------------- | -------- | ---- | ---- |
+| Content-Type | application/json | 是       |      |      |
+
+**Body**
```

Markdownlint (0.35.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)